packages/components/src/components/post-accordion/post-accordion.tsx

```diff
@@ -16,13 +18,34 @@ export class PostAccordion {
 
   @Element() host: HTMLPostAccordionElement;
 
+  /**
+   * Defines the hierarchical level of the `post-accordion-item` headers within the headings structure.
+   */
+  @Prop() readonly headingLevel?: HeadingLevel;
```

Comment:
I did not make it required to avoid a breaking change.

Reply:
Should we add a default value here? I mean, there was a default value (2), when we had the prop on the post-accordion-item.

Reply:
The post-accordion-item still has a default value that will be used if the property is not defined on the post-accordion.
Do you mean having the default property for the docs?

Reply:
Does this make sence, our would it be better to get the default value allways from the post-accordion component?

post-accordion headingLevel (default value of 2, not required)
post-accordion-item headingLevel (no default value, required)

Would this make more sence? Would it be a breaking change? What do you think?

Reply:
Removing the default title level from the post-accordion element would be a drastic change in my opinion.
We can always add a default to the post-accordion too, but that would be difficult to maintain because the two defaults need to be in sync. Also, it wouldn't fit [this idea](https://github.com/swisspost/design-system/pull/3067#discussion_r1598583123).